README.md

```diff
@@ -33,30 +33,48 @@ We plan to add OAuth 2 support shortly.
 
 ## Getting started
 
-ApiCenter consists of a RESTful backend service written in Kotlin and a SPA frontend in Angular. Both are contained in this git repo.
+ApiCenter consists of a RESTful backend service written in Kotlin and a SPA frontend in Angular. Both are contained in this git repo. There are two ways to run ApiCenter: one combines the tasks of the frontend and backend together, for simplicity; the other requires two terminal sessions, however has advantages for developers.
```

Comment:
"however it has advantages for developers"? And maybe we should also say what the advantages are ;)

Reply:
Yeah. Maybe just change the sentence here that usually, you want the single build, but you can do it separately for development?